com.onlyoffice.DesktopEditors.appdata.xml

```diff
@@ -0,0 +1,32 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<application>
+  <id type="desktop">com.onlyoffice.DesktopEditors</id>
+  <name>ONLYOFFICE Desktop Editors</name>
+  <developer_name>ONLYOFFICE</developer_name>
+  <summary>Office productivity suite</summary>
+  <metadata_license>CC0-1.0</metadata_license>
+  <project_license>LicenseRef-AGPL-3.0</project_license>
+  <url type="homepage">https://onlyoffice.com</url>
+  <icon type="remote" height="256" width="256">https://res.cloudinary.com/canonical/image/fetch/f_auto,q_auto,fl_sanitize,w_72,h_72/https://dashboard.snapcraft.io/site_media/appmedia/2018/01/icon_IzCN8xA.png</icon>
+  <description>
+    <p>Create, view and edit text documents, spreadsheets and presentations of any size and complexity. Work on documents of most popular formats: DOCX, ODT, XLSX, ODS, CSV, PPTX, ODP, etc. Deal with multiple files within one and the same window thanks to the tab-based user interface.</p>
+  </description>
+  <screenshots>
+    <screenshot type="default">
+      <image type="source">https://static-www.onlyoffice.com/v9.5.0/images/mainpage/may2018/editors/document-short.jpg</image>
+      <caption>Main window</caption>
+    </screenshot>
+  </screenshots>
+  <kudos>
+    <kudo>HiDpiIcon</kudo>
+  </kudos>
+  <content_rating type="oars-1.1" />
+  <categories>
+    <category>Productivity</category>
+    <category>Office</category>
+  </categories>
+  <releases>
+    <release version="5.2.8" date="2018-02-14"/>
```

Comment:
Update to v5.4.1

Reply:
Seems the website reports this as 5.2.8. Might be a oversight however. Does the application report itself as 5.4.1?

![Uploading image.png…]()

Reply:
![Captura de tela de 2019-10-31 22-02-00](https://user-images.githubusercontent.com/26097419/67995614-1c971480-fc2a-11e9-9a31-b6ea52ccb485.png)